apps/wallet/src/ui/app/components/recovery-passphrase/RecoveryPassphrase.tsx

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Info16 } from '@mysten/icons';
+import { useState } from 'react';
+
+import { Text } from '_app/shared/text';
+import { HideShowDisplayBox } from '_components/HideShowDisplayBox';
+import { MenuLayout } from '_components/menu/content/MenuLayout';
+import { PasswordInputDialog } from '_components/menu/content/PasswordInputDialog';
+import { useNextMenuUrl } from '_components/menu/hooks';
+import { useAppDispatch } from '_hooks';
+import { loadEntropyFromKeyring } from '_redux/slices/account';
+import { entropyToMnemonic, toEntropy } from '_shared/utils/bip39';
+
+export function RecoveryPassphrase() {
+    const [passwordConfirmed, setPasswordConfirmed] = useState(false);
+    const [mnemonic, setMnemonic] = useState('');
```

Comment:
Can we change this to be an array of strings like we do in Backup page https://github.com/MystenLabs/sui/blob/main/apps/wallet/src/ui/app/pages/initialize/backup/index.tsx#L51? To void having the whole mnemonic in one string? and make `HideShowDisplayBox` render it in spans?